apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/server/S3JsonMessageSource.java

```diff
@@ -20,11 +20,21 @@
  * @param <T> the type parameter
  */
 public class S3JsonMessageSource<T> implements MessageSource<T> {
+  /** The S3 object to read. */
```

Comment:
This is nice; provides a nice operational model for creating a S3RifMessageSource to stream RIF files for ETL.